packages/@react-stately/color/src/Color.ts

```diff
@@ -237,7 +237,7 @@ class RGBColor extends Color {
       case 'red':
       case 'green':
       case 'blue':
-        return {minValue: 0, maxValue: 255, step: 1, pageSize: 0x10};
+        return {minValue: 0x0, maxValue: 0xFF, step: 0x1, pageSize: 0xF};
```

Comment:
I think default `pageSize` for RGB colors should be 0x10 or 16, even though 0xFF or 255 is evenly divisible by 0xF, or 15. The hex colors at each step make more sense that way.

Reply:
I struggled with that for a long time, I figured even division was more important than anything else. Adobe Color increments by 0x1a, which is even weirder. So I figure no one is actually expecting anything in particular. But this brings the most consistent experience, it's predictable going forward and backward from either end.

Reply:
The old school web developer in me favors incrementing and snapping to an 8-bit value for RGB, 16 rather than 15.

Reply:
Hmmm, 0x11 (17) seems to produce even better results than 0xF (15) or 0x10 (16); it divides 255 evenly, and every third color when incrementing or decrementing a channel is "web safe," ie #330000, as opposed to #2D0000 or #300000 for 0xF and 0x10 respectively.

Reply:
```suggestion
        return {minValue: 0x0, maxValue: 0xFF, step: 0x1, pageSize: 0x11};
```

Reply:
That makes sense to me, I'll update it